### PR TITLE
Redo the interface's help menu for Project Athena.

### PR DESCRIPTION
**Pull request by MarcusLlewellyn**
Redo the help menu so that:
- The about menu has been updated
  - The top image now shows Project Athena
  - Added Project Athena Contributor copyright
  - Removed EOS blockchain reference
  - Added attribution for the Opus codec.
- The other help menu links have been updated
  - Documentation links now point to Athena hosted sites
  - Bug reporting now points to the Athena GitHub issue tracker
  - Removed link to forums until we actually have one

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
"An open-source virtual reality platform."
…
"open source" instead of "open-source"?

**Comment by ctrlaltdavid** [comment]
Help > Controls Reference dialogs have "support@highfidelity.com" down the bottom.

**Comment by MarcusLlewellyn** [comment]
> Help > Controls Reference dialogs have "[support@highfidelity.com](mailto:support@highfidelity.com)" down the bottom.
> 
> And "Report Problem" top right opens e-mail to [support@highfidelity.com](mailto:support@highfidelity.com)/

Yeah. That dialog contains what is basically an HTML slideshow of images. There's no actual text (strings) there to edit. I was going to tackle it in a separate PR.

File: interface/resources/qml/hifi/dialogs/TabletAboutDialog.qml [hunk 1]
```diff
@@ -93,17 +93,31 @@ Rectangle {
...
+            }
+        }
         Item { height: 20; width: 1 }
         RalewayRegular {
-            textFormat: Text.StyledText
-            linkColor: "#00B4EF"
             color: "white"
-            property string link: "https://eos.io/"
-            text: "Blockchain technology from <a href=\"" + link + "\">EOS</a>."
+            text: "© 2019 - 2020 Project Athena Contributors"
```
[the remark this answers is not in the document]
Comment (MarcusLlewellyn) [reply interface/resources/qml/hifi/dialogs/TabletAboutDialog.qml]: Done. :)

**Pull request merged**